python/paddle_fl/paddle_fl/examples/generate_job_from_program/README.md

```diff
@@ -0,0 +1,155 @@
+# Example to Load Program from a Pre-defined Model
+
+This document introduces how to load a pre-defined model, and transfer into program that is usable by PaddleFL.
+
+### Dependencies 
+
+- paddlepaddle>=1.8
+- paddle_fl>=1.0
+
+Please use pip which has paddlepaddle installed
+
+```sh
+pip install paddle_fl
+``` 
+
+### Compile Time
+
+#### How to save a program
+
+In program_saver.py, you can defind a model. And save the program in to 'load_file'
+
+```python
+input = fluid.layers.data(name='input', shape=[1, 28, 28], dtype="float32")
```

Comment:
import paddle.fluid as fluid?

Reply:
import codes are added in all example README files